db/functions/graphql.go

```diff
@@ -150,6 +150,11 @@ ResolverLoop:
 				multiError = multiError.Append(err)
 				break ResolverLoop // stop
 			}
+			if config.MaxCodeSize != nil && len(fnConfig.Code) > *config.MaxCodeSize {
```

Comment:
This block should go earlier, before the resolver gets compiled, so we don't try to compile a huge chunk of code. At line 134 would be good (just after validating `fnConfig.args`.
Also, don't break out of ResolverLoop; that prevents discovering any more errors. (The MaxResolverCount check does break out of the loop because once there are too many resolvers there's no point looking at any more.)